<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Daves WMS Tile Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="zoomStart">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>48</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="zoomEnd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>48</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WMS URL</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sridSelector">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>4326</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27700</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>900913</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>SRID</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>621</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="makeTilesButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Tiles</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusMessages">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>521</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>http://localhost/cgi-bin/mapserv.ngx?map=/home/daveb/Development/JavascriptAPITest/raster_mapfile.map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuNew"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_Tiles">
   <property name="text">
    <string>Create Tiles</string>
   </property>
  </action>
  <action name="actionDelete_Tiles">
   <property name="text">
    <string>Delete Tiles</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeTilesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>makeTiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>makeTiles()</slot>
 </slots>
</ui>
